formats/financial_release/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  // "required": [
```

Comment:
I'm assuming these comments are here just for reference purposes. I don't believe you're going to need any links for financial releases themselves, so I believe if you simply don't include the links file everything will still build and validate correctly. See https://github.com/alphagov/govuk-content-schemas/blob/master/bin/combine_publisher_schema#L39 for details.